### DISCUSSION TITLE
Display Issue with Frozen Columns

### DISCUSSION THREAD

**Discussion by ibrahimGit123** [serenity-is/Serenity#6702] [Help]
There is a glitch when we scroll in frozen columns, the data in the left side frozen columns becomes blurred out. See below:

With issue
![image](https://user-images.githubusercontent.com/57414714/221019396-77b8d3e4-080c-4e9c-a87b-047260e4e1e9.png)

With out issue
![image](https://user-images.githubusercontent.com/57414714/221019478-f219539f-6c73-46b1-a2df-242a96ba38d7.png)

Not sure if it is related to this issue https://github.com/serenity-is/Serenity/issues/1959

My Frozen columns implementation code
```
       protected getSlickOptions() {
            var opt = super.getSlickOptions();
            opt.frozenColumns = 7;
            opt.layoutEngine = new Slick.FrozenLayout();
            return opt;
        }

```

**Comment by Julian1993ARG** [comment]
Has you resolve it? i am trying use frozen but i get this issue.
A.FrozenLayout is not a constructor
